ChaosMod/Effects/Condition/EffectCondition.h

```diff
@@ -3,17 +3,20 @@
 #include "Util/Logging.h"
 #include "Util/MacroConcat.h"
 
+#include <unordered_map>
+#include <functional>
+
 enum class EffectConditionType
 {
 	None,
 	VotingEnabled, // Voting is enabled
 	ProportionalVotingEnabled
 };
 
-#define REGISTER_EFFECT_CONDITION(conditionType, condition)                           \
-	namespace                                                                         \
-	{                                                                                 \
-		EffectCondition CONCAT(_effectCondition, __LINE__)(conditionType, condition); \
+#define REGISTER_EFFECT_CONDITION(conditionType, condition, description)                           \
+	namespace                                                                                      \
```

Comment:
IMO "description" is non-descriptive, should rather be called "failReason" or something of the sort

Reply:
Sadly I am very bad at naming things.